src/plugins/displayLayout/components/BoxView.vue

```diff
@@ -21,15 +21,18 @@
  *****************************************************************************/
 
 <template>
-    <layout-frame :item="item"
-                  :grid-size="gridSize"
-                  @move="(gridDelta) => $emit('move', gridDelta)"
-                  @endMove="() => $emit('endMove')">
-        <div class="c-box-view"
-             :style="style">
-        </div>
-    </layout-frame>
- </template>
+<layout-frame
+    :item="item"
```

Comment:
Also, why did the <layout-frame> indentation change?

Reply:
baseIndent was set to 0 in "vue/html-indent" rule